gojira.sh

```diff
@@ -537,7 +537,7 @@ function image_name {
   if [[ -n $GOJIRA_IMAGE ]]; then return; fi
 
   # No supplied dependency versions
-  if [[ -z $LUAROCKS || -z $OPENSSL || -z $OPENRESTY ]]; then
+  if [[ -z $LUAROCKS || (-z $OPENSSL && -z $BORINGSSL) || -z $OPENRESTY ]]; then
```

Comment:
This is creating a subshell which may cause unknown issues; WDYT about this instead?

```suggestion
  if [[ -z $LUAROCKS || -z "${OPENSSL}${BORINGSSL}" || -z $OPENRESTY ]]; then
```

Since this was using the `&&` operator this could be combined into a single check without resorting to a subshell.